index.html

```diff
@@ -273,6 +307,11 @@
     })
   }
 
+  function submitAmount(form) {
+    calculator.amount = parseInt(form.querySelector('[name=amount]').value, 10)
+  }
+
+  calculator.init()
```

Comment:
I overrides the value on the next round, yet does not reset for now not to complexity the code.
Suggestions are welcome to do so ;)